test/integration/pause_test.go

```diff
@@ -0,0 +1,88 @@
+// +build integration
+
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"context"
+	"os/exec"
+	"testing"
+)
+
+func TestPause(t *testing.T) {
+	MaybeParallel(t)
+
+	type validateFunc func(context.Context, *testing.T, string)
+	profile := UniqueProfileName("pause")
+	ctx, cancel := context.WithTimeout(context.Background(), Minutes(30))
+	defer CleanupWithLogs(t, profile, cancel)
+
+	// Serial tests
```

Comment:
optional - but IMHO, this test would be more readable if it was simplified into a single test-case.

Reply:
I think when something fails it's much nicer to see what failed and also when someone reads the report they can see what functionalities failed and what passed.

Reply:
For example, if someone see we test a functionality named
"TestDeletePaused" they know why we pause and the. Delete in one or and when we have a test report we can link to only what failed